types/evidence_test.go

```diff
@@ -421,4 +460,23 @@ func TestEvidenceVectors(t *testing.T) {
 		hash := tc.evList.Hash()
 		require.Equal(t, tc.expBytes, hex.EncodeToString(hash), tc.testName)
 	}
+
+	// testCases2 := []struct {
```

Comment:
I'd be inclined to either fix or delete (or use `t.Skip()` so the compiler continues to run)

Reply:
This was the main reason the PR is still in 'draft' mode :) deleted!